button/generic-data-pipeline
# Populating new exchange rates table.

Populate new exchange rates table that can be looked up against for ad-hoc queries while also being used for populating USD values in existing tables.

Testing includes:
1. Bring up local mysql db.
2. Source the proper environment variables.
3. Run db-migrate if you don't have the chartio db locally.
4. Run python manage.py exchange_rates then confirm the table reflects as expected.
